woocommerce/woocommerce-gateway-stripe
# feat: add webhook status message

# Changes proposed in this Pull Request:
I noticed that in the "old" settings we show the webhook status information.
But we don't do that in the "new" settings (and in the designs).

Old settings:
![Screen Shot 2021-10-07 at 1 41 57 PM](https://user-images.githubusercontent.com/273592/136443694-31b443ee-a8f6-4976-bcae-22804e7445c2.png)


Adding it to the account section, for now.

With some status
![Screen Shot 2021-10-07 at 1 59 30 PM](https://user-images.githubusercontent.com/273592/136446108-b729410a-3afa-492c-bd8f-2a82a650baad.png)

When the webhook secret is missing
![Screen Shot 2021-10-07 at 1 58 08 PM](https://user-images.githubusercontent.com/273592/136445947-1e800c3e-9b6c-4f63-8e65-7fdf88bf6f19.png)


When everything in the world goes wrong:
![Screen Shot 2021-10-07 at 1 58 53 PM](https://user-images.githubusercontent.com/273592/136446025-25b7b22b-ce0b-47eb-8e65-575e002b6a68.png)


# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Ensure your account is connected
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings
- The "Account" section now shows the webhook status
- Since we don't have a way to edit the webhook secret, you can do so in the "old" settings, with the `_wcstripe_feature_upe_settings` flag disabled
